rate-limiting-global/pom.xml

```diff
@@ -36,10 +36,11 @@
             <Bundle-Activator>io.stargate.db.limiter.global.GlobalRateLimitingActivator</Bundle-Activator>
             <Import-Package><![CDATA[
               org.osgi.framework,
-              io.stargate.db
+              io.stargate.core.*,
+              io.stargate.db,
+              io.stargate.db.*,
             ]]></Import-Package>
-            <Export-Package>io.stargate.db.limiter.global</Export-Package>
```

Comment:
Doesn't this need to be exported in the event someone wants to use it?

Reply:
The linkage should be done via the proper API, I think. That is `RateLimitingManager` exported by `persistence-api`.

Linking to an implementation class in a bundle is probably not a good idea. If we want to share implementaiton code, I believe it would be preferable to publish it as a plain jar and have it _embedded_ in the bundle that needs to use it.